java/libs.javacapi/nbproject/org-netbeans-libs-javacapi.sig

```diff
@@ -1,2849 +1,3 @@
 #Signature file v4.1
-#Version 8.48.0
-
```

Comment:
this file is now empty. intended?

to disable sigtest this could be put into project properties: `sigtest.skip.gen=true`